docs/source/jupyterhub_spawners.md

```diff
@@ -0,0 +1,8 @@
+# Fixtures for JupyterHub Spawners
+
+## Module: {mod}`pytest_jupyterhub.jupyterhub_spawners`
+
+```{eval-rst}
```

Comment:
I don't think I understand why the [`eval-rst`](https://myst-parser.readthedocs.io/en/latest/syntax/roles-and-directives.html#how-directives-parse-content) directive is used here. I don't think we have any rst content anywhere?

Reply:
It is because of [this instruction here](https://github.com/jupyterhub/autodoc-traits#use-with-myst-parser) when using a markdown source file instead of rst. Or I may have misunderstood the instruction.

Reply:
This is what the docs look like without it:

![docs2](https://user-images.githubusercontent.com/114513578/216998993-2b5520ef-f130-4077-b25e-be73e9317362.PNG)

Reply:
> It is because of [this instruction here](https://github.com/jupyterhub/autodoc-traits#use-with-myst-parser) when using a markdown source file instead of rst. Or I may have misunderstood the instruction. It was the only thing that made the build work locally.

Oh, right! I was the one getting things the other way around. Sorry for the confusion. 👍🏼